cats/__init__.py

```diff
@@ -182,6 +183,12 @@ def positive_integer(string):
         type=positive_integer,
         help="Amount of memory used by the job, in GB",
     )
+    parser.add_argument(
+        "-n",
+        "--no-colour",  # or could use American spelling like many other CLIs?
```

Comment:
can we accept colour and color? I think add_argument accepts any number of names so just adding "--no-color" ought to work.

Reply:
Good idea, I've implemented this now, so marking as resolved.